### PR TITLE
correct parameters

### PR DESCRIPTION
**Pull request by c0ldcalamari**

### CODE REVIEW

File: resource-synchronizer/poet_ready.py [hunk 1]
```diff
@@ -32,12 +32,12 @@ def rollback(code_dir, rollback_dir):
             dest_path_file = path_file.replace(rollback_dir, code_dir)
             shutil.copy2(path_file, dest_path_file)
 
-    cleanup()
+    cleanup(rollback_dir)
 
 
 def main():
     code_dir = Path(sys.argv[0]).resolve(strict=True)
```
Comment (rnathuji) [inline resource-synchronizer/poet_ready.py:39]: Should this be updated to `argv[1]` as well?

**Review by rnathuji — COMMENTED** [review]
I think the tests are failing due to [this line](https://github.com/openstax/content-synchronizer/blob/e04c05fdce7e1bbba6a61a859b38982e17b74a16/resource-synchronizer/test_sync_scripts.py#L585). Try making that `["", repo_dir]` and it should work.

File: resource-synchronizer/poet_ready.py (hunk 1 above)
Comment (c0ldcalamari) [reply resource-synchronizer/poet_ready.py]: aiy yes, thank you!

**Review by rnathuji — APPROVED** [review]

**Comment by c0ldcalamari** [comment]
Testing: PR is tested with [content-synchronizer PR#17](https://github.com/openstax/content-synchronizer/pull/17#issuecomment-853279514)

**Pull request merged**